tasks/pkg-windows-opts.yml

```diff
@@ -1,19 +1,19 @@
 ---
 - name: Set DD Username Arg
   set_fact:
-    win_install_args: "{{ win_install_args }} DDAGENTUSER_NAME={{ datadog_windows_ddagentuser_name }}"
+    agent_win_install_args: "{{ agent_win_install_args }} DDAGENTUSER_NAME={{ datadog_windows_ddagentuser_name }}"
```

Comment:
This variable is defined in `defaults/main.yml`, but I don't see it respected here. We'll have to first set `agent_win_install_args` to value of `win_install_args`.